react/features/video-status-label/components/VideoStatusLabel.js

```diff
@@ -93,10 +130,16 @@ export class VideoStatusLabel extends Component {
                 ? t('videoStatus.hd') : t('videoStatus.sd');
         }
 
-        const filmstripClassName
+        // Determine which classes should be set on the component. These classes
+        // will used to help with animations and setting position.
+        const baseClasses = 'video-state-indicator moveToCorner';
+        const filmstrip
             = _filmstripVisible ? 'with-filmstrip' : 'without-filmstrip';
+        const oneOnOne = _remoteVideosVisible && _remoteVideosCount
+            ? 'with-remote-videos' : 'without-remote-videos';
```

Comment:
I thinking remoteVideosCount was an intermediary step I had used to move the label to the top right. It shouldn't be needed.